### PR TITLE
Show selected items in rm msg

### PR DESCRIPTION
**Pull request by luukvbaal**
Close https://github.com/jarun/nnn/issues/1479

### CODE REVIEW

**Review by N-R-K — APPROVED** [review]
I remember a couple times I trashed something else in another directory by mistake because they were still selected. So I think showing the selection count makes sense.

File: src/nnn.c [hunk 1]
```diff
@@ -1429,7 +1429,8 @@ static char confirm_force(bool selection)
 
 	snprintf(str, 64, messages[MSG_FORCE_RM],
 		 g_state.trash ? utils[UTIL_GIO_TRASH] + 4 : utils[UTIL_RM_RF],
-		 (selection ? "selection" : "hovered"));
+		 (selection ? xitoa(nselected) : "hovered"),
```
Comment (jarun) [inline src/nnn.c:1432]: Why not something like:

- hovered
- selected 5

**Review by jarun — CHANGES_REQUESTED** [review]

File: src/nnn.c (hunk 1 above)
Comment (luukvbaal) [reply src/nnn.c]: Sure, done.

**Comment by jarun** [comment]
Thank you!

**Pull request merged**